tests/Fakes/Models/IgnoredFieldsPost.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace OrisIntel\AuditLog\Tests\Fakes\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use OrisIntel\AuditLog\Traits\AuditLoggable;
+
+class IgnoredFieldsPost extends Model
+{
+    use AuditLoggable;
+
+    protected $guarded = [];
+
+    protected $table = 'posts';
+
+    public function getAuditLogIgnoredFields() : array
```

Comment:
would it make sense for this to be passed in as a const with an array?

Reply:
ex: const AUDIT_IGNORED_FIELDS = ['posted_at', ...]

Reply:
Like similar to `$guarded` ?

Reply:
I kind of prefer the model overriding method as it allows you to use dynamic logic of what you want to ignore. With this we could still use a constant on our own implementations and just use it internally on a base class.

Reply:
@ericnkatz yeah, a var is probably a better way than const
@tomschlick That makes sense to me, thinking about it more, I could see a situation where a field may be ignored under certain conditions but not others. A function would allow you to specify that logic